go.mod

```diff
@@ -54,14 +54,14 @@ require (
 	github.com/ti-mo/conntrack v0.4.0
 	github.com/vishvananda/netlink v1.1.1-0.20211101163509-b10eb8fe5cf6
 	github.com/vmware/go-ipfix v0.5.12
-	golang.org/x/crypto v0.0.0-20220214200702-86341886e292
+	golang.org/x/crypto v0.0.0-20220427172511-eb4f295cb31f
 	golang.org/x/exp v0.0.0-20200224162631-6cc2880d07d6
-	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
-	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
+	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
+	golang.org/x/net v0.0.0-20220906165146-f3363e06e74c
+	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
```

Comment:
do we know why Dependabot (https://github.com/antrea-io/antrea/blob/main/.github/dependabot.yml) is not taking care of upgrading these?

Reply:
maybe Dependabot uses a different tool to scan CVEs, I used [trivy](https://github.com/aquasecurity/trivy)

Reply:
Dependabot is supposed to open PRs to update our dependencies automatically, irrespective of CVEs

Reply:
Maybe because the versions are not tags so they are excluded? otherwise dependabot would upgrade them once for every patch that committed to their code.